h2o-core/src/main/java/water/Weaver.java

```diff
@@ -54,6 +54,8 @@
     } catch( NotFoundException nfe ) { throw new RuntimeException(nfe); }
   }
 
+  public static ClassPool getPool() { return _pool; }
```

Comment:
I agree 👍  the thinking here is that dynamically loaded classes will want to be inserted into the classpath so that weaver can fish them out of the pool later. The _pool is actually already published by virtue of being the default pool, and perhaps it's fine to just use defaultPool().

One might do this for example:

Weaver.getPool().insertClassPath(...);  // allows _pool to change

the other option, which has the same effect, is to simply ask for the default pool:

ClassPool.getDefault().insertClassPath(...);  // depends on _pool being the default pool


I am fine with either option